scripts/validate/validate_rest_proxy.sh

```diff
@@ -3,7 +3,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
 ################################## GET KAFKA CLUSTER ID ########################
-KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
+KAFKA_CLUSTER_ID=$(curl -s -k https://localhost:8091/v1/metadata/id --cert scripts/security/mds.certificate.pem --key scripts/security/mds.key --tlsv1.2 --cacert scripts/security/snakeoil-ca-1.crt | jq -r ".id")
```

Comment:
Is client key and cert needed here?

Reply:
@mikebin update after making other changes in the certs: I believe this can be simplified to:

```
curl https://localhost:8091/v1/metadata/id --tlsv1.2  --cacert scripts/security/snakeoil-ca-1.crt
```

Is this consistent with your understanding?

Reply:
Yes, that looks good to me.